test/integration.js

```diff
@@ -198,8 +198,8 @@ describe('integration tests:', function () {
       });
 
       it('should execute a long command with arguments', function (done) {
-        exec('very complicated deep command abc123 -rad -sleep \'well\' -t -i \'j\' ', done, function () {
-          stdout().should.equal('radtjabc123');
+        exec('very complicated deep command abc123 -rad --sleep \'well\' -t -i \'j\' ', done, function () {
```

Comment:
Must have been someone else's PR that I didn't notice. That's fine - I'll fix it. The rest of your stuff looks fine.